README.md

```diff
@@ -854,6 +855,139 @@ registration: {
 ```
 - **registration.click** - Function that is called when the registration button is clicked
 
+
+## Registration
+
+```javascript
+    var signIn = new OktaSignIn({
+      baseUrl: 'https://acme.okta.com',
+      clientId: '{{myClientId}}',
+      registration: {
+        parseSchema: function(schema, onSuccess, onFailure) {
+           // handle parseSchema callback
+        },
+        preSubmit: function (postData, onSuccess, onFailure) {
+           // handle preSubmit callback
+        },
+        postSubmit: function (response, onSuccess, onFailure) {
+            // handle postsubmit callback
+        }
+      },
+      features: {
+         registration: true
+      }
+    });
+```
+
+Following configurations are needed to enable registration on a self hosted sign in widget
+
+
+- **features.registration:** Used to enable registration feature on the widget.
+    ```javascript
+    // https://github.com/okta/okta-signin-widget#feature-flags
+    features.registration: 'true'
+
+    ```
+- **clientId:** Client Id pre-registered with Okta for the OIDC authentication flow
+
+    ```javascript
+    clientId: 'GHtf9iJdr60A9IYrR0jw'
+    ```
+Optional configurations :
+
+- **parseSchema:** Callback used to mold the JSON schema that comes back from Okta API.
+
+    ```javascript
+    /**
+     The callback function is passed 3 arguments:
+     @params : schema, onSuccess, onFailure
+     1) @param {Object} schema: json schema returned from the API.
+     2) @param {Function} onSuccess: success callback.
+     3) @param {Function} onFailure: failure callback. Note: accepts a errorObject that can be used to show form level or field level errors.
+    **/
+    parseSchema: function (schema, onSuccess, onFailure) {
+      // This example will add an additional field to the registration form
+        schema.profileSchema.properties.address = {
+          'type': 'string',
+          'description': 'Street Address',
+          'default': 'Enter your street address',
+          'maxLength': 255
+        };
+        schema.profileSchema.fieldOrder.push('address');
+        onSuccess(schema);
+    }
+    onFailure callback accepts a errorObject that can be used to show form level or field level errors
```

Comment:
We can remove this line, since we note it above.

Reply:
Fixed